ropetest/advanced_oi_test.py

```diff
@@ -20,6 +20,7 @@ def tearDown(self):
         super(DynamicOITest, self).tearDown()
 
     def test_simple_dti(self):
+        self.project.prefs['perform_doa'] = True
```

Comment:
for the doi tests it seems sensible to just do this in the setup directly instead of seemingly each test